Cython/Utility/Coroutine.c

```diff
@@ -809,6 +809,29 @@ PyObject *__Pyx_Coroutine_FinishDelegation(__pyx_CoroutineObject *gen) {
     return ret;
 }
 
+static PyObject *__Pyx_PyGen_Send(PyObject *gen, PyObject *arg) {
+#if PY_VERSION_HEX < 0x030A00A1
```

Comment:
I think this version check is wrong as the changes that require this are post 10a1 and pre 10a2 (which does not exist yet).

Reply:
Thanks for reviewing.
Is it possible to know 3.10+ or not?